tests/queries/0_stateless/02029_optional_error.sql

```diff
@@ -0,0 +1,22 @@
+DROP TABLE IF EXISTS t;
+
+CREATE TABLE t ( a Int32 ) ENGINE = Memory;
+
+INSERT INTO t VALUES (abs(functionThatDoesNotExists(42))); -- { clientError? UNKNOWN_FUNCTION }
+INSERT INTO t VALUES (abs(functionThatDoesNotExists(42))); -- { clientError? }
+INSERT INTO t VALUES (abs(functionThatDoesNotExists(42))); -- { clientError }
+INSERT INTO t VALUES (1, 2); -- { clientError? SYNTAX_ERROR }
+INSERT INTO t VALUES (1, 2); -- { clientError? }
+INSERT INTO t VALUES (1, 2); -- { clientError }
+INSERT INTO t VALUES (1); -- { clientError? UNKNOWN_FUNCTION }
+INSERT INTO t VALUES (1); -- { clientError? SYNTAX_ERROR }
```

Comment:
But it seemed that `?` means that we can either expect no error or exactly error after `?`. Why we allow two different errors?

Reply:
Yes, this query are succeed, so both of codes will be ignored and do not affect anything.